lib/cli.mjs

```diff
@@ -526,6 +563,64 @@ export default async function main(argv) {
 		options.computeEditInfo = computeEditInfo;
 	}
 
+	if (isBoolean(pruneSuppressions)) {
+		options.pruneSuppressions = pruneSuppressions;
+	}
+
+	if (isBoolean(suppressAll)) {
+		options.suppressAll = suppressAll;
+	}
+
+	if (isString(suppressionsLocation)) {
+		options.suppressionsLocation = suppressionsLocation;
+	}
+
+	if (Array.isArray(suppressRule) && suppressRule.length > 0) {
+		options.suppressRule = suppressRule;
+	} else {
+		options.suppressRule = undefined;
+	}
```

Comment:
question

Can't you add an `undefined` default to https://github.com/stylelint/stylelint/pull/8564/files#diff-d1470268bb8c3a42692b2c5249d7d31832ceeed1b1f16fbdbc814f8013c36b31R348-R351?

Reply:
Even if I specify `default: undefined`, it still returns an empty array.
 
> Note: If used in conjunction with isMultiple, the default flag value is set to [].

See: [booleanDefault - sindresorhus/meow: 🐈 CLI app helper](https://github.com/sindresorhus/meow?tab=readme-ov-file#booleandefault)